service/history/workflow/context.go

```diff
@@ -985,28 +987,15 @@ func (c *ContextImpl) UpdateRegistry(ctx context.Context) update.Registry {
 	}
 
 	if c.updateRegistry == nil {
-		nsIDStr := c.MutableState.GetNamespaceEntry().ID().String()
-
+		ns := c.MutableState.GetNamespaceEntry().Name()
```

Comment:
Drive-by fix: this was "namespace ID" but should be "namespace name"

Reply:
![](https://user-images.githubusercontent.com/2232524/75076254-832f8f00-54b4-11ea-81b9-1a6648a486a5.png)